VMEncryption/test/test_resource_disk_util.py

```diff
@@ -0,0 +1,199 @@
+#!/usr/bin/env python
+#
+# *********************************************************
+# Copyright (c) Microsoft. All rights reserved.
+#
+# Apache 2.0 License
+#
+# You may obtain a copy of the License at
+# http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing
+# permissions and limitations under the License.
+#
+# *********************************************************
+
+""" Unit tests for the ResourceDiskUtil module """
+
+import unittest
+import mock
+
+from main.ResourceDiskUtil import ResourceDiskUtil
+from main.DiskUtil import DiskUtil
+from main.Common import CommonVariables
+from console_logger import ConsoleLogger
+
+
+class TestResourceDiskUtil(unittest.TestCase):
+    def setUp(self):
+        self.logger = ConsoleLogger()
+        self.mock_disk_util = mock.create_autospec(DiskUtil)
+        self.mock_passhprase_filename = "mock_passphrase_filename"
+        mock_public_settings = {}  # update this before tests
```

Comment:
in reference to the "# update this before tests" comment - is this a to-do item?  maybe some commented samples of what to set this to if necessary

Reply:
I think I was expecting to use it during tests, but it turned out ended up not needing to modify it.
I'll just remove the comment.